WakaTime/ConfigFile.swift

```diff
@@ -8,15 +8,22 @@
 import Foundation
 
 struct ConfigFile {
+    private static var filePath: String {
+        NSString.path(withComponents: FileManager.default.homeDirectoryForCurrentUser.pathComponents + [".wakatime.cfg"])
+    }
+
+    private static var filePathInternal: String {
+        NSString.path(withComponents: FileManager.default.homeDirectoryForCurrentUser.pathComponents + [".wakatime-internal.cfg"])
+    }
 
-    static func getSetting(section: String, key: String) -> String {
-        let file = NSString.path(withComponents: FileManager.default.homeDirectoryForCurrentUser.pathComponents + [".wakatime.cfg"])
+    static func getSetting(section: String, key: String, internalConfig: Bool = false) -> String? {
```

Comment:
I changed the return type to `String?` and adapted existing code to handle `nil` cases.

Reply:
We should check for both empty string `""` and `nil`, I think this works:

`(string ?? "").isEmpty`

Seen in 461f352e71865a84373d75235f29ffe859e0f2ff.